src/main/java/emu/grasscutter/game/entity/gadget/platform/BaseRoute.java

```diff
@@ -38,22 +39,15 @@ public static BaseRoute fromSceneGadget(SceneGadget sceneGadget) {
     }
 
     public boolean startRoute(Scene scene) {
-        if (this.isStarted) {
-            return false;
-        }
         this.isStarted = true;
         this.isActive = true;
-        this.startSceneTime = scene.getSceneTime()+300;
+        this.startSceneTime = scene.getSceneTime();
```

Comment:
I think the +300 was reqruied for some syncronisation stuff with the albedo elevator, so does it still work with this?

Reply:
Maybe with abilities this is fixed?

Reply:
What sort of synchronization stuff? Does this look good?

https://github.com/Anime-Game-Servers/Grasscutter-Quests/assets/1877986/f8df840e-94d4-43ce-8f33-9af1e09c5594

Reply:
not fully sure what it was originally, but it looks good, yeah